client-sdk/ts-web/core/docs/getting-started.md

```diff
@@ -1,4 +1,95 @@
-# Putting together the pieces
+# Getting started
+
+## Overview
+
+1. Getting this SDK and building
+1. Connecting from your web app
+
+## Getting this SDK and building
+
+```sh
+npm install @oasisprotocol/client@alpha
+```
+
+Install this package [from
+npm](https://www.npmjs.com/package/@oasisprotocol/client).
+
+**TODO: Update this command when this library gets out of alpha.**
+
+You'll need a bundler.
+We have [a sample that uses webpack](../playground/webpack.config.js).
+
+## Connecting from your web app
+
+```js
+import * as oasis from '@oasisprotocol/client';
+
+// Use https://grpc-web.oasis.dev/api/testnet to interact with the testnet instead.
+const client = new oasis.OasisNodeClient('https://grpc-web.oasis.dev/api/mainnet');
+```
+
+This connects to a public oasis-node instance.
+For security and performance reasons, some methods are not permitted.
+See the Advanced deployment section below for how to set up your own node with
+a custom set of enabled methods.
+
+# Example code
+
+We have [a few sample interactions](../playground/src/index.js).
+
+Not all methods used this sample are permitted on the public oasis-node
```

Comment:
```suggestion
Not all methods used in this sample are permitted on the public oasis-node
```

Reply:
oh shoot I missed this one

Reply:
https://github.com/oasisprotocol/oasis-sdk/pull/181